resources/js/Components/Table/DMEntryTable/DMEntryTable.tsx

```diff
@@ -48,8 +48,8 @@ const DMEntryTable = ({data}: DMEntryPropType) => {
         {
             property: 'character',
             title: 'Character',
-            render: (value: string) => (
-                <Chip label={value ?? 'Unassigned'} variant='outlined' />
+            render: (value: any) => (
+                <Chip label={value?.name ?? 'Unassigned'} variant='outlined' />
```

Comment:
so this means that if there is no value for character, it will be `Unassigned` ?

Reply:
Yes, that's what will show up on the DM Entries table.